lib/marquee.dart

```diff
@@ -530,6 +534,7 @@ class _MarqueeState extends State<Marquee> with SingleTickerProviderStateMixin {
 
   Future<bool> _scroll() async {
     await _makeRoundTrip();
+    if (isDone) widget.onDone!();
```

Comment:
The `onDone` callback may be `null`.
```suggestion
    if (isDone && widget.onDone != null) {
      widget.onDone!();
    }
```